readers/cassandra/messages.go

```diff
@@ -128,7 +128,9 @@ func buildQuery(chanID string, rpm readers.PageMetadata) (string, []interface{})
 	if err != nil {
 		return condCQL, vals
 	}
-	json.Unmarshal(meta, &query)
+	if err = json.Unmarshal(meta, &query); err != nil {
```

Comment:
Don't shadow the original `err`: `if err := json.Unmarshal(meta, &query); err != nil {`.
Also, please check the later usage (L43 of this file) and if potential error should be handled.